File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/FillTextCommand_View.cpp

#include "FillTextCommand_View.h"
#include "FillTextCommand_View.h"
#include <vector>
#include <string>

namespace commands
{
  std::string FillTextCommand_View::getMyValueTextBoxText( ) 
  {
    return this->MyValueTextBoxText;
  }
  std::vector<std::string> FillTextCommand_View::getMyFreeValueComboBoxComboBoxEntries( ) 
  {
    return this->MyFreeValueComboBoxComboBoxEntries;
  }
  std::string FillTextCommand_View::getMyFreeValueComboBoxSelectedEntry( ) 
  {
    return this->MyFreeValueComboBoxSelectedEntry;
  }
  std::string FillTextCommand_View::getMyFreeValueComboBoxText( ) 
  {
    return this->MyFreeValueComboBoxText;
  }
  void FillTextCommand_View::myValueTextFilled(std::string text ) 
  {
    
  }
  void FillTextCommand_View::myFreeValueTextFilled(std::string text ) 
  {
    
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/FillTextCommand_View.cs
/// <filename>
///     FillTextCommand_View.cs
/// </filename>
using System.Collections.Generic;

namespace commands
{
    internal class FillTextCommand_View
  {
        private string MyValueTextBoxText;
     
        public     virtual string getMyValueTextBoxText()
    {
      return this.MyValueTextBoxText;
    }
     
        private System.Collections.Generic.List<string> MyFreeValueComboBoxEntries = new System.Collections.Generic.List<string>();
     
        public     virtual System.Collections.Generic.List<string> getMyFreeValueComboBoxEntries()
    {
      return this.MyFreeValueComboBoxEntries;
    }
     
        private string MyFreeValueComboBoxSelectedEntry;
     
        public     virtual string getMyFreeValueComboBoxSelectedEntry()
    {
      return this.MyFreeValueComboBoxSelectedEntry;
    }
     
        private string MyFreeValueComboBoxText;
     
        public     virtual string getMyFreeValueComboBoxText()
    {
      return this.MyFreeValueComboBoxText;
    }
     
        public     virtual void myValueTextFilled(string text)
    {
      
    }
     
        public     virtual void myFreeValueTextFilled(string text)
    {
      
    }
  }
}

